resources/views/auth/verify-email.blade.php

```diff
@@ -0,0 +1,28 @@
+<x-guest-layout>
+  <div class="mx-auto max-w-96 p-8 text-center text-on-background-1">
+    <x-p class="mt-2" size="2xl">Дякуюмо за реєстрацію до</x-p>
+    <a class="mx-auto inline-block cursor-pointer" href="{{ url('/') }}"><x-logo withText /></a>
```

Comment:
We have some example with text with logo? Just interest how it looks

Reply:
![image](https://github.com/gvary-ua/web-reader/assets/36802078/ce755e33-c838-4fbd-aa5c-d5850c126f8d)